app/Resources/translations/messages.sv.xliff

```diff
@@ -518,6 +518,31 @@
                     Best regards,
                     Kompisbyrån</target>
             </trans-unit>
+            <trans-unit id="match.email.user">
+                <source>match.email.user</source>
+                <target>Hello %user_name%!
+
+Great that you want to be part of Kompisbyrån!
```

Comment:
@jongotlin I will do this right away after it is deployed. We will do the trial with english for now and use the old system until we have the swedish in place (it is already written)